pkg/cli/admin/upgrade/status/status.go

```diff
@@ -261,7 +261,7 @@ func (o *options) Run(ctx context.Context) error {
 		if message = progressing.Message; message == "" {
 			message = "<none>"
 		}
-		fmt.Fprintf(o.Out, "The cluster version is not updating (%s=%s).\n\n  Reason: %s\n  Message: %s\n", progressing.Type, progressing.Status, reason, strings.ReplaceAll(message, "\n", "\n  "))
+		fmt.Fprintf(o.Out, "The cluster is not updating.")
```

Comment:
We should include a newline in the output IMO.

```suggestion
		fmt.Fprintf(o.Out, "The cluster is not updating.\n")
```

Reply:
correct. just verified it prints % in stdout

Reply:
Addressed :+1: